src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -337,7 +337,7 @@ private function parseDefinition(\DOMElement $service, $file, array $defaults)
         }
 
         foreach ($this->getChildren($service, 'call') as $call) {
-            $definition->addMethodCall($call->getAttribute('method'), $this->getArgumentsAsPhp($call, 'argument', $file));
+            $definition->addMethodCall($call->getAttribute('method'), $this->getArgumentsAsPhp($call, 'argument', $file), $call->getAttribute('use-result'));
```

Comment:
I don't think `getAttribute` returns a bool which is what `addMethodCall` expects. So you need to add a type casting and use `XmlUtils::phpize` to proper support `'false'`